src/background.js

```diff
@@ -354,7 +356,14 @@ function loadUrl(status) {
 
 function main() {
   console.info('Fetching releases from: ' + LATEST_RELEASES_URL)
-  axios.get(LATEST_RELEASES_URL).then(async result => {
+  const existVersionFile = fs.existsSync(
```

Comment:
Remove redeclaration of `existVersionFile` [line 393](https://github.com/witnet/sheikah/pull/1720/files#diff-16f022ec89282b37542e3374910a5e4a4cf37596abf8f723f0d7f8ab1096f853R393)